fdbserver/workloads/SpecialKeySpaceCorrectness.actor.cpp

```diff
@@ -1217,92 +1217,6 @@ struct SpecialKeySpaceCorrectnessWorkload : TestWorkload {
 				wait(tx->onError(e));
 			}
 		}
-		{
```

Comment:
This test invokes broken special key-space behavior that is being fixed in [another PR](https://github.com/apple/foundationdb/pull/6831).